.golangci.yaml

```diff
@@ -20,14 +21,11 @@ linters:
   - asciicheck
   - containedctx
   - decorder
-  - depguard
-  - dogsled
```

Comment:
We have a lot of ignored parameters around

Reply:
That sounds like a reason to maybe keep it. Let me check what changed from the previous version.

Reply:
Errors are checked elsewhere, but for some things it seems the returns are ignored on purpose

Reply:
(errcheck checks return errors that are ignored)

Reply:
If you are running the linter locally, are you passing `--new-from-rev=75173a17cf`?

Reply:
Readed `dogsled` and added the same commit we updated in https://github.com/openshift/release/pull/44928/files